src/Dapp/dapp.js

```diff
@@ -133,18 +201,9 @@ export default class Dapp extends Component {
       hash = `#/${params.details}`;
     }
 
-    return (
-      <iframe
-        className={ styles.frame }
-        frameBorder={ 0 }
-        id='dappFrame'
-        name={ name }
-        onLoad={ this.onDappLoad }
-        sandbox='allow-forms allow-popups allow-same-origin allow-scripts allow-top-navigation'
-        scrolling='auto'
-        src={ `${src}${hash}` }
-      />
-    );
+    return isElectron()
+      ? this.renderWebview(src, hash)
+      : this.renderIframe(src, hash);
```

Comment:
This could probably go?

Reply:
It could be useful for dev. I.e. with this we can still develop in browser. Less resource-hungry maybe?